tests/unit/has.ts

```diff
@@ -112,9 +126,21 @@ registerSuite({
 				}, TypeError);
 			},
 
-			'feature is already defined; returns false'() {
-				assert.isTrue(hasAdd(feature, true));
-				assert.isFalse(hasAdd(feature, false));
+			'throws when feature is already defined'() {
+				let err: Error;
+				assert.isTrue(hasAdd('abc', true));
+				try {
+					assert.isFalse(hasAdd('abc', false));
+				} catch (e) {
+					err = e;
+				}
+				assert.isTrue(err instanceof Error);
+				assert.equal(err.message, 'abc already exists and no overwrite flag was passed');
+			},
+
+			'feature name is lowercased'() {
```

Comment:
Do you also need a test that includes a test that when passing an uppercase string to `has` it lowercased?